README.md

```diff
@@ -51,14 +47,13 @@ from within a virtual environment (e.g., via [virtualenv](https://pypi.org/proje
 
 1. Clone the code from GitHub
 2. `python setup.py install --user`
-2. Run `propka31` with a .pdb file (see Examples)
+2. Run `propka32` with a .pdb file (see Examples)
```

Comment:
I would only use `propka32` as the executable name if different propka versions can be installed side-by-side. But since the Python module's name is just `propka`, that's currently not the case. I think `propka` would be better as the executable name.

Reply:
I agree with you but the comments in the code (https://github.com/Electrostatics/propka-3.1/commit/573b38db386b1e79ed4053a1bcc4ee2b516ff142) have indicated that there have been problems with having the script and the module share the same name.  I can't reproduce this but am hesitant to break something.  What do you think?

Reply:
Ah yes, if you have a launch script named `propka.py`, that would lead to a conflict if you do `python path/to/propka.py` because the script directory is prepended to `PYTHONPATH` by default. On Unix, launch scripts typically have no extension and are regular shell executables, e.g. consider `/usr/bin/pip` and the `pip` Python module, they don't conflict. AFAIK, you don't even this script because [`entry_points`](https://github.com/jensengroup/propka-3.1/blob/master/setup.py#L53) creates one for you (on Windows it creates a small `.exe` I think).

If you're unsure, you can keep the `propka32` name of course (or use `propka3` as a middle ground :-))